cluster-autoscaler/cloudprovider/gce/templates.go

```diff
@@ -215,6 +215,8 @@ func BuildGenericLabels(ref GceRef, machineType string, nodeName string, os Oper
 	}
 	result[apiv1.LabelZoneRegion] = ref.Zone[:ix]
 	result[apiv1.LabelZoneFailureDomain] = ref.Zone
+	result[apiv1.LabelZoneRegionStable] = ref.Zone[:ix]
```

Comment:
nit: I'd prefer to have the stable version right after the non-stable one (see LabelArch and LabelArchStable above).